lighthouse-core/audits/uses-rel-preload.js

```diff
@@ -0,0 +1,97 @@
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('./audit');
+const Util = require('../report/v2/renderer/util');
+const UnusedBytes = require('./byte-efficiency/byte-efficiency-audit');
+
+class UsesRelPreloadAudit extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Performance',
+      name: 'uses-rel-preload',
+      description: 'Preload key requests',
+      helpText: 'Consider using <link rel=preload> to prioritize fetching late-discovered ' +
+        'resources sooner [Learn more](https://developers.google.com/web/updates/2016/03/link-rel-preload).',
+      requiredArtifacts: ['devtoolsLogs'],
+      scoringMode: Audit.SCORING_MODES.NUMERIC,
+    };
+  }
+
+  static _flattenRequests(chains, maxLevel, minLevel = 0) {
+    const requests = [];
+    const flatten = (chains, level) => {
+      Object.keys(chains).forEach(chain => {
+        if (chains[chain]) {
+          const currentChain = chains[chain];
+          if (level >= minLevel) {
+            requests.push(currentChain.request);
+          }
+
+          if (level < maxLevel) {
+            flatten(currentChain.children, level + 1);
+          }
+        }
+      });
+    };
+
+    flatten(chains, 0);
+
+    return requests;
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    const devtoolsLogs = artifacts.devtoolsLogs[UsesRelPreloadAudit.DEFAULT_PASS];
+
+    return Promise.all([
+      artifacts.requestCriticalRequestChains(devtoolsLogs),
+      artifacts.requestMainResource(devtoolsLogs),
+    ]).then(([critChains, mainResource]) => {
+      const results = [];
+      let totalWastedMs = 0;
+      // get all critical requests 2 levels deep
+      const criticalRequests = UsesRelPreloadAudit._flattenRequests(critChains, 2, 2);
+      criticalRequests.forEach(request => {
+        const networkRecord = request;
+        if (!networkRecord._isLinkPreload && networkRecord.protocol !== 'data') {
+          // calculate time between mainresource.endTime and resource start time
+          const wastedMs = (request._startTime - mainResource._endTime) * 1000;
+          totalWastedMs += wastedMs;
+          results.push({
+            url: request._url,
+            downloadTime: Util.formatMilliseconds(wastedMs),
+          });
+        }
+      });
+      const headings = [
+        {key: 'url', itemType: 'url', text: 'URL'},
+        {key: 'downloadTime', itemType: 'url', text: 'Time to download'},
```

Comment:
wastedMs is duration from end of document request to start of these critical ones. so i wouldn't call it Time to Download.. hmmm.  

> Potential Savings

and let's keep `wastedMs` as the property name.